content/participants/ttjc@HarshitBharani/index.md

```diff
@@ -0,0 +1,40 @@
+---
+title: Harshit Bharani
+type: "participant"
+name: "Harshit Bharani"
+portfolio: ""
+github-link: "https://github.com/HarshitBharani"
+linkedin-link: ""
+looking-for: "job/internship"
+description: "I am a student from govt engineering college . learning react and javascript"
+---
+
+Hi! I am a Harshit Bharani let me walk you through my intro, projects and blogs.
+
+### introduction
+
+learning to code
+
+## why are you taking this challenge?
+
+
+I want to learn.
+and also want a job 
+
+## something interesting about you?
+
+I know lyrics to almost all Bollywood songs.
+
+### help I need?
+
+
+
+### how to contact me?
+
+- instagram
```

Comment:
provide links to your contact details
```
[Description](Link goes here)
```
use proper markdown.

Reply:
added link for instagram and linkdin